optimism/src/keccak/interpreter.rs

```diff
@@ -428,7 +428,7 @@ pub trait KeccakInterpreter<F: One + Debug + Zero> {
                                 Some(q),
                             ),
                     );
-                    self.constrain(
+                    /*self.constrain(
```

Comment:
Commenting out this constraint because it is failing in the prover side (it holds in witness constraint check though). Needs further investigation from my side.